### ISSUE TITLE
Files opening in floaterm instead of main window

### ISSUE THREAD

**Issue by sergiza** [ptzz/lf.vim#35]
I'm using rifle as my file opener, i'm guessing that's what's caussing the issue.
This is in my lfrc
`cmd open $set -f; rifle -p 0 $fx`
`set ifs "\n"`

![Screenshot_20220823_163242](https://user-images.githubusercontent.com/25955239/186186458-466f9a75-4989-4297-8467-2230575bc0a0.png)

**Comment by ptzz** [comment]
Does it work with an empty lfrc?

I tried with your lfrc and it works for me. Are you sure you're executing the `open` command (i.e. not `edit`) from lf?

**Comment by sergiza** [comment]
Works as intended with an empty lfrc, and works as intended with everything else except those two lines (rifle).
And yea, i've tried open, edit, open with. None works.

**Comment by ptzz** [comment]
Ok, I don't know what's wrong. For me, when running lf with `-last-dir-path` and `-selection-path` arguments (i.e. as a file selector), the selected/opened file is just written to the specified path and any action associated with open in lfrc is ignored.

Does it work if you use e.g. `cat` instead of `rifle`? Maybe try remove the `set ifs` part as well.

**Comment by sergiza** [comment]
`set ifs "\n"` has nothing to do with it from what i've tested now. It's all in the `cmd open $set -f; rifle -p 0 $fx` line. I don't know what you mean by using `cat` instead of `rifle`, if you give me a line of what you mean i'll test it.
I also noticed that when I "solve" the problem, it opens in a new terminal instead of opening in my current terminal. So maybe it's just hopeless.

**Comment by sergiza** [comment]
![Peek_2022-08-24](https://user-images.githubusercontent.com/25955239/186405358-2fc786b6-3535-41ad-86ae-cba2dc57b83e.gif)

**Comment by ptzz** [comment]
Try with only this in the lfrc and check if it still misbehaves:

```
cmd open ${{
    cat $f
}}
```

> I also noticed that when I "solve" the problem, it opens in a new terminal instead of opening in my current terminal. So maybe it's just hopeless.

You mean with an empty lfrc, the file gets opened in a new vim session in another terminal..?

**Comment by sergiza** [comment]
> You mean with an empty lfrc, the file gets opened in a new vim session in another terminal..?

Exactly this, I recorded a gif but it doesn't look clear enough, let me try again.

**Comment by ptzz** [comment]
I see now..

**Comment by sergiza** [comment]
![2terminals](https://user-images.githubusercontent.com/25955239/186406621-95655372-d664-4116-b928-9661b07ef92d.gif)

**Comment by sergiza** [comment]
> Try with only this in the lfrc and check if it still misbehaves:
> 
> ```
> cmd open ${{
>     cat $f
> }}
> ```

Looks like it does the command inside of floaterm too.

![cat](https://user-images.githubusercontent.com/25955239/186407593-6b524cde-591e-4f35-83c1-111aac2173e5.gif)

**Comment by ptzz** [comment]
That's super weird, it should do nothing except return to originating vim session with the selected file opened. No clue what's going on. I'm on a Mac with `lf --version` 27, NVIM v0.7.2.

How do you start Lf, just `:Lf` command?

**Comment by sergiza** [comment]
I'm using the default map `map <leader>f :Lf<CR>`.
The only thing I changed is
```
if exists('g:lf_command_override')
  let s:lf_command = g:lf_command_override
else
  let s:lf_command = '~/.config/lf/lf'
endif
```
My lf is an appimage, so I needed to change that. Latest version lf r27 and nvim v0.7.2 too. Also I'm on Kubuntu 22.04.1 LTS.

**Comment by ptzz** [comment]
Could you try this branch https://github.com/ptzz/lf.vim/tree/fix-custom-lf-path? It seems the introduction of floaterm broke custom path to lf.

**Comment by sergiza** [comment]
> Could you try this branch https://github.com/ptzz/lf.vim/tree/fix-custom-lf-path? It seems the introduction of floaterm broke custom path to lf.

It fixed it! For now, it seems like everything works as intended.
Thank you for taking the time to help.

**Comment by ptzz** [comment]
👍
